docs/use-github-in-bcgov/github-enterprise-user-licenses-bc-government.md

```diff
@@ -32,52 +34,52 @@ For more information on our GitHub organizations and their uses, see [B.C. gover
 - [Access the GitHub Enterprise account](#access-the-github-enterprise-account)
 - [Pay for the GitHub Enterprise user licences](#pay-for-the-github-enterprise-user-licences)
 
-## Benefits of a GitHub Enterprise user licence
-
-The enterprise user licence offers various features over the Free and Teams version. To find out more on the differences in features and pricing, see [Pricing](https://github.com/pricing).
+## B.C. government GitHub Enterprise Overview
 
-## Enterprise account ownership
+GitHub Enterprise offers various features beyond those of the Free and Teams tiers. To find out more about the features of GitHub Enterprise, you may wish to review [GitHub's product page](https://github.com/enterprise).
 
-The GitHub Enterprise account is owned by OCIO and managed by the Platform Services team. Ministries own GitHub organizations linked to the B.C. government’s GitHub Enterprise accounts and the private repositories within their organizations.  
+There are certain notable differences between the [public facing `bcgov` GitHub organization](https://github.com/bcgov) and B.C. government's GitHub Enterprise environment. For example, the ability to create public repositories is disabled within GitHb Enterprise since with B.C. government public open source code should be in the [public `bcgov` organization](https://github.com/bcgov). 
 
-User licenses for accessing the private repositories come from a central pool under the GitHub Enterprise account and must be purchased for each user that needs access to the private repositories in the ministry’s private organizations.  Management of this pool of seats is done in partnership with each sector or ministry purchasing their own allocated seat count.
+Other details of GitHub Enterprise include the following:
 
-When your private organization is linked to the GitHub Enterprise account certain settings are automatically enabled or disabled. For example, the ability to create public repositories within this private organization is removed to fall in line with all Provincial open-source code remaining in `bcgov`. Other features include the following:
+* Single sign-on is provided via Azure AD (aka Entra ID)
+* Login uses government-issued IDIR identities
+* Users will be automatically assigned new GitHub IDs, distinct from public GitHub IDs 
```

Comment:
Users will be automatically assigned new GitHub IDs, changing login-in flow for teams using both public and private repositories